app/views/organisations/_promotional_features.html.erb

```diff
@@ -0,0 +1,22 @@
+<section class="organisations__margin-bottom organisations__promotions">
+  <% @documents.promotional_features.each do |feature| %>
+    <div class="<%= feature[:parent_column_class] %>">
+      <%= render "govuk_publishing_components/components/heading", {
+        text: feature[:title],
+        heading_level: 2,
+        padding: true,
+        border_top: 5,
+        brand: @organisation.brand,
+        font_size: 19
```

Comment:
These appear smaller than the other section headings on the page - is this intended?

Reply:
It is, although maybe it shouldn't be. I noticed that a lot of the headings on the Civil Service page were too long and wrapping onto the next line unnecessarily, making the layout look wonky.

Reply:
Have changed - we shouldn't be developing to fix content.